lib/practice_exercises.rb

```diff
@@ -1,13 +1,48 @@
-
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: (On)
```

Comment:
Since `delete_at` is an O(n) method (because it shifts subsequent elements over one index), your full method has a time complexity of O(n<sup>2</sup>)